example_test.go

```diff
@@ -210,17 +194,15 @@ func ExampleBatchPostgres() {
 func ExampleBatchMySQL() {
 	params := [][]any{
 		{42, true, "str"},
-		{69, true, "noice"},
+		{69, true, "noise"},
```

Comment:
no, it's `noice` 😆 https://www.youtube.com/watch?v=h3uBr0CCm58

Reply:
:D got it, reverted